numpy/distutils/tests/test_misc_util.py

```diff
@@ -78,4 +78,4 @@ def test_get_shared_lib_extension(self):
 def test_installed_npymath_ini():
     # Regression test for gh-7707.  If npymath.ini wasn't installed, then this
     # will give an error.
-    info = get_info('npymath')
+    get_info('npymath')
```

Comment:
Would be great to add some basic test of `info` here, rather than just ignoring the return value